modules/virt-about-qemu-guest-agent-web.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-information.adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual Machine Overview* tab and the *Details* tab display information about the *Hostname*, *Operating system*, *Time zone*, and *Logged in users*.
+
+The *Virtual Machine Overview* shows information about the guest operating system installed on the virtual machine.
```

Comment:
Is **Virtual Machine Overview** a tab or a page, or what?

Reply:
Tab.